src/bin/crates-admin.rs

```diff
@@ -53,3 +53,9 @@ fn main() -> anyhow::Result<()> {
         Command::EnqueueJob(command) => enqueue_job::run(command),
     }
 }
+
+#[test]
+fn verify_cli() {
+    use clap::CommandFactory;
+    Command::command().debug_assert();
```

Comment:
I was curious what `debug_assert()` is doing. It looks like it runs all of https://docs.rs/clap_builder/4.4.8/src/clap_builder/builder/debug_asserts.rs.html, but only when `debug_assertions` is enabled, which should be the case when running the test suite. In other words, `debug_assert()` apparently enables some additional checks that won't even be caught when running the binary from a release build.

tl;dr LGTM 👍